### DISCUSSION TITLE
RSS'24 demo problems with launching

### DISCUSSION THREAD

**Discussion by botmayank** [aerostack2/aerostack2#656] [Q&A]
Hi, I tried replicating the tutorial from the workshop at home and faced this issue, will spend more time on it to debug once I have spare cycles but was wondering if I'm missing something obvious. @miferco97 

https://github.com/aerostack2/rss24_demo/issues/2

**Comment by miferco97** [comment]
No more comments have come in, so I’ll go ahead and close this one as solved 🙂
Don’t hesitate to reopen if anything else comes up!
